src/app/globals.css

```
@@ -85,3 +85,7 @@
   animation: slow-spin 18s linear infinite;
   transform-origin: center;
 }
+
+body {
+  font-family: 'Roboto', sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)